src/lib/api-client.js

```diff
@@ -184,3 +202,14 @@ export function getModelsForNewItem() {
         hashHistory.push('/items/new');
     });
 }
+export function deleteItem(address){
+    return del('item', {
+        itemAddress: address
+    }).then(data => {
+        Dispatcher.handleAction('ITEMS_RECEIVED', data);
```

Comment:
I don't think it's blocking either, but for the future, I think it is in the scope of this PR to call the action ITEM_DELETED since we know we will need it.  IMO, adding the action call in the toast story is out of scope of that story.  Granted, it's only one line, but the toast story should only be adding the handlers to the toast store that respond to the actions, not adding actions calls to api-client.